contracts/src/v0.8/dev/automation/2_1/Chainable.sol

```diff
@@ -10,7 +10,7 @@ contract Chainable {
   /**
    * @dev addresses of the next contract in the chain **have to be immutable/constant** or the system won't work
    */
-  address public immutable i_next;
+  address private immutable i_next;
```

Comment:
curious why change to private

Reply:
there's already a public getter for this val